contents/founders/startup-ops-toolkit.md

```diff
@@ -67,37 +71,37 @@ We use [Fondo](https://www.tryfondo.com/) (US only) and they are unbelievably go
+[Pry](http://pry.co/) (now part of Brex) has been a game-changer for helping us build our forecasts. Yes, we all know you can “just” do it in Excel (which you will want to do if you have a finance background), but a tool like Pry makes it so much easier to build and share your plan with non-finance people. And you can skip the 2 years chained to your desk at Goldman Sachs! You can start with their cheapest plan at $45/mo. 
 
-Pry connects seamlessly with your accounting software (Quickbooks, Xero) as well as most banks, and you then just fill out assumptions manually or using simple formulae. It also has neat dashboards for visualizing everything, comparing budget vs. actuals, and scenario modeling.
+Pry connects with your accounting software (Quickbooks, Xero) as well as most banks, and you then just fill out assumptions manually or using simple formulae. It also has neat dashboards for visualizing everything, comparing budget vs. actuals, and scenario modeling. I love Pry so much that I actually [wrote a guide](/founders/startup-finance-without-finance) about how we use it to run finance at PostHog - without a finance person. 
 
 Be responsive when it comes to re-forecasting – don't just rigidly stick to an arbitrary annual or quarterly process because it's 'standard practice'. We review budget vs. actuals monthly, and then reforecast as and when it feels appropriate. 
 
 ### Where to put the money
 
 Put it in [Brex](https://www.brex.com) (US only). Brex also makes it incredibly easy to manage virtual credit cards, and the built-in expense management negates the need for something like Expensify. It doesn’t charge you anything. They make distributing your cash between a current account and MMFs (basically low-risk interest bearing funds) very easy too.
 
-> You might want a backup account in case Brex goes down, a la Silicon Valley Bank. In the beginning, Brex should be fine as your deposit is insured up to $6m. Beyond that, you may want an account with a more boring bank as backup. Bear in mind that the onboarding process will be totally horrific, consisting of 70-page forms and onboarding calls. 
+> You might want a backup account in case Brex goes down, à la Silicon Valley Bank. In the beginning, Brex _should_ be fine as your deposit is insured up to $6m. Beyond that, you may want an account with a more boring bank as backup. Bear in mind that the onboarding process will be totally horrific, consisting of 70-page forms and onboarding calls. We actually tried this with JP Morgan Chase and then reverted back to [Mercury](https://mercury.com/) because the experience was so bad. 
 
 In Europe, [Revolut](https://www.revolut.com/) is about the only digital bank that gets anywhere near Brex features-wise, although I find its platform a lot buggier. We're basically waiting for Brex to launch in Europe so we can shut down our Revolut account as they're not actually a real bank in the UK (so no deposit protection). 
```

Comment:
```suggestion
In Europe, [Revolut](https://www.revolut.com/) is about the only digital bank that gets anywhere near Brex features-wise, although I find its platform a lot buggier. We're basically waiting for Brex to launch in Europe so we can shut down our Revolut account as they're not actually a real bank in the UK (so no deposit protection). 

> **Addendum:** You might want a backup account in case Brex goes down, à la Silicon Valley Bank. In the beginning, Brex _should_ be fine as your deposit is insured up to $6m. Beyond that, you may want an account with a more boring bank as backup. Bear in mind that the onboarding process will be totally horrific, consisting of 70-page forms and onboarding calls. We actually tried this with JP Morgan Chase and then reverted back to [Mercury](https://mercury.com/) because the experience was so bad. 
```

Would prefer boxout at the end of the section as a boxout in the middle may as well just be its own paragraph without the boxout.